selfdrive/controls/lib/events.py

```diff
@@ -615,6 +615,11 @@ def wrong_car_mode_alert(CP: car.CarParams, sm: messaging.SubMaster, metric: boo
                               audible_alert=AudibleAlert.chimeDisengage),
   },
 
+  EventName.processNotRunning: {
+    ET.NO_ENTRY: NoEntryAlert("Not All Processes Are Running",
```

Comment:
what about something like system malfunction? I think this has too much information for a user and can cause confusion. what's a process? will they start up in a minute if I try to reenagage?

Reply:
Something like: "System Malfunction. Reboot Device"?